.claude/skills/adversary/scripts/serve-dashboard.py

```
@@ -0,0 +1,214 @@
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Adversary — Live Dashboard Server"
+    )
+    parser.add_argument(
+        "--port", type=int, default=8450,
+        help="Port to serve on (default: 8450)"
+    )
+    parser.add_argument(
+        "--report", type=str, default="/tmp/groundwork-report.json",
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/groundwork-report.json"

(S108)